docs/framework/server-side-editor-api.md

```diff
@@ -0,0 +1,264 @@
+- **Automated Review Systems**: Build systems that automatically review and suggest content changes, like grammar checks or style improvements
+- **AI-powered Editing**: Make automated suggestions while users are actively editing, helping improve content quality
+- **Automated Revision Control**: Track and manage document versions automatically, perfect for maintaining content history and audit trails
+- **Automated Publishing**: Prepare and process content for publication, including formatting, metadata updates, and resolving comments
+- **Custom Integration**: Connect the editor with your existing systems and workflows, such as CMS or document management systems
+- **Automatic Checkpoints**: Create automatic checkpoints in your document
+
+## Getting Started with Server-Side Editor API
+
+This guide shows you how to write scripts that can be executed through the Server-Side Editor API endpoint. The following sections provide examples of such scripts, each demonstrating a specific use case that can be automated on the server side.
+
+For information about setting up and using the endpoint itself, see the {TODO: link Cloud Services Server-side Editor API} documentation.
+
+## Working with Content
+
+### Simple Text Changes
+
+Here's a simple example. Imagine you're working on a document and need to change every instance of "entirely" to "completely". Instead of doing it manually, you can do it with one line of code:
+
+```js
+// Replace all instances of "entirely" with "completely" in the document.
+editor.execute( 'replaceAll', 'completely', 'entirely' );
+```
+
+This one line will:
+- Find all instances of "entirely" in your document
+- Change each one to "completely"
+- Keep all the formatting around the changed text
+
+This is perfect for fixing typos, updating old terms, or making any bulk changes to your documents.
+
+### Updating Links in Your Document
+
+Now, consider a more complex scenario. You need to update all links in your document from `/docs/` to `/documents/`. This is a common task when moving content between environments or updating your site structure.
+
+```js
+// Get the root element and create a range that covers all content.
+const root = editor.model.document.getRoot();
+const range = editor.model.createRangeIn( root );
+const items = Array.from( range.getItems() );
+
+// Use model.change to ensure proper undo/redo support.
+editor.model.change( writer => {
+    for ( const item of items ) {
+        let href = item.getAttribute( 'linkHref' );
+
+        if ( item.is( 'textProxy' ) && href ) {
+            // Update the link URL.
+            href = href.replace( '/docs/', '/documents/' );
+            writer.setAttribute( 'linkHref', href, item );
+        }
+    }
+} );
+```
+
+This approach is particularly useful when you need to:
+- Update many links at once
+- Fix broken links across your content
+- Change link patterns in bulk
```

Comment:
Following my previous suggestions, I think that here I'd link to some guide in framework to learn API basics? Or just not write anything at all.